tools/xchain-suite/src/pages/SwapPage.tsx

```
@@ -306,9 +306,9 @@ export default function SwapPage() {
           destinationAddress,
           protocol: selectedQuote.protocol,
           slippageToleranceBps: slippageBps,
-          // Streaming swap parameters (not applicable for Chainflip)
+          // Streaming swap parameters (only for THORChain/MAYAChain)
           ...(isStreaming &&
-            selectedQuote.protocol !== 'Chainflip' && {
+            selectedQuote.protocol !== 'Chainflip' && selectedQuote.protocol !== 'OneClick' && {
```

ESLint

[error] Insert `⏎···········`

(prettier/prettier)